tests/fixtures/features/modified-place-wikidata.json

```diff
@@ -0,0 +1,131 @@
+      "type": "Feature",
+      "properties": {
+        "building": "yes",
+        "place": "New York",
+        "wikidata": "Q60",
+        "boundary": "administrative"
+      },
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [-124.49707031249999,
+              37.19533058280065
+            ],
+            [-124.49707031249999,
+              41.04621681452063
+            ],
+            [-120.62988281249999,
+              41.04621681452063
+            ],
+            [-120.62988281249999,
+              37.19533058280065
+            ],
+            [-124.49707031249999,
+              37.19533058280065
+            ]
+          ]
+        ]
+      }
+    },
+    "newVersion": {
+      "type": "Feature",
+      "properties": {
+        "place": "New York",
+        "wikidata": "Q70",
+        "boundary": "administrative"
+      },
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [-124.49707031249999,
+              37.19533058280065
+            ],
+            [-124.49707031249999,
+              41.04621681452063
+            ],
+            [-120.62988281249999,
+              41.04621681452063
+            ],
+            [-120.62988281249999,
+              37.19533058280065
+            ],
+            [-124.49707031249999,
+              37.19533058280065
+            ]
+          ]
+        ]
+      }
+    },
+    "expectedResult": {
+      "result:modifiedPlaceWikidata": true
+    }
+  }, {
+    "Description": "This a modification of wikidata tag for features that have no place tag.Should not be flagged",
+    "oldVersion": {
+      "building": "yes",
+      "wikidata": "Q60"
+    },
+    "geometry": {
+      "type": "Polygon",
+      "coordinates": [
+        [
+          [-118.36797058582306,
+            33.81124046079714
+          ],
+          [-118.36797058582306,
+            33.81136971841298
+          ],
+          [-118.36779892444609,
+            33.81136971841298
+          ],
+          [-118.36779892444609,
+            33.81124046079714
+          ],
+          [-118.36797058582306,
+            33.81124046079714
+          ]
+        ]
+      ]
+    },
+    "newVersion": {
+      "type": "Feature",
+      "properties": {
+        "building": "yes",
+        "wikidata": "Q70"
+      },
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [-118.36797058582306,
+              33.81124046079714
+            ],
+            [-118.36797058582306,
+              33.81136971841298
+            ],
+            [-118.36779892444609,
+              33.81136971841298
+            ],
+            [-118.36779892444609,
+              33.81124046079714
+            ],
+            [-118.36797058582306,
+              33.81124046079714
+            ]
+          ]
+        ]
+      }
+    },
+    "expectedResult": {
+      "result:modifiedPlaceWikidata": false
```

Comment:
This looks great. Just in case when there is false result by compare function , we return result as `{}`. So this can be modified to `"expectedResult" : {}` . Also to test whether your all test case pass or not, you can do `node tests/test_compare_function.js tests/fixtures/features/modified-place-wikidata.json` . 

Just noted that you have placed your fixture file in `features` directory, which could be stored directly placed in `tests/fixtures/`  so as to maintain consistency with other fixture files.